cockatrice/src/client/tabs/tab_deck_editor.cpp

```diff
@@ -673,12 +673,12 @@ void TabDeckEditor::retranslateUi()
     deckMenu->setTitle(tr("&Deck Editor"));
 
     cardInfoDock->setWindowTitle(tr("Card Info"));
-    deckDock->setWindowTitle(tr("Deck"));
+    deckDock->setWindowTitle(tr(ZONE_DECK));
```

Comment:
That would change the capitalisation of the displayed string.

Reply:
Yeah good catch, this doesn't need to be replaced